README.md

```diff
@@ -50,56 +50,15 @@ To grab dependencies, build, and run the backend server, from the top-level dire
 ```
 ./gradlew run
 ```
-This builds and runs the backend server. 
+This builds and runs the backend server.
 
 The backend server exposes the `localhost:4567/ulcrs` resource. For instance, to get tutor information for all tutors, hit the `localhost:4567/ulcrs/tutor/` endpoint with a get request to get all tutors, or `localhost:4567/ulcrs/tutor/{id}` to get the tutor with the specified id.
 
 ### Electron UI
-To grab dependencies and run the UI, from the top-level directory, run:
-``` 
-
-npm start
-```
-This will pull in dependencies and display the Electron application. 
-
-### Java Server
-**Dependencies:** You must have Java 8 and Gradle installed. 
-
-There are two ways of interacting with the backend server:
-1) Use IntelliJ to manage and run Gradle tasks (including building, running, and testing the server) 
-2) Interact with the ULCRS backend server from the command line
-
-Option 1 should be straightforward using Gradle, so we discuss Option 2, how to interact with the ULCRS backend server from the command line:
-
-**Build:**
+To run the UI, from the electron-src/ under the top-level directory, run:
```

Comment:
I would say: "To run the UI, from the top-level directory, run: ```cd electron-src \n npm start```"

Reply:
Cant have newlines in comments but basically change the sentence and add the `cd electron-src` in the "code" section with the three backticks (i.e. ```)